### DISCUSSION TITLE
How to implement jQuery?

### DISCUSSION THREAD

**Discussion by wcoy7** [filamentphp/filament#11065] [Help]
### Package

Panel builder

### Package Version

v3.2.14

### How can we help you?

I searched and couldn't find anything.  My app uses various jQuery based UI elements.  I followed [Laracoding.com](https://laracoding.com/how-to-install-and-use-jquery-with-laravel-vite/).  In my site I see that the jQuery is in app.js but jQuery, $, etc... are undefined in the console.

Any ideas?

**Comment by leandrocfe** [comment]
Filament uses Livewire + [AlpineJs](https://alpinejs.dev/)
IMO, you should refactor it. However, you can register JS files
https://filamentphp.com/docs/3.x/support/assets#registering-javascript-files

**Comment by wcoy7** [comment]
Not sure how that answers the question.

— Warren
